pkgs/misc/drivers/canon-cups-capt/capt.nix

```
@@ -0,0 +1,156 @@
+{
+  stdenv, fetchzip,
+  autoconf,
+  automake,
+  libtool,
+  glib,
+  gnome2,
+  pkgconfig,
+  libxml2,
+  cndrvcups-common
+}:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  pname = "cndrvcups-capt";
+  version = "2.71";
+
+  src = fetchzip {
+    url = "http://gdlp01.c-wss.com/gds/6/0100004596/05/linux-capt-drv-v271-uken.tar.gz";
+    sha256 = "0agpai89vvqmjkkkk2gpmxmphmdjhiq159b96r9gybvd1c1l0dds";
+  };
+
+  patches = [ ./0001-patch-missing-include.patch ];
+
+  unpackPhase = ''
+    mkdir -p ${name}
+
+    tar -xzf $src/Src/${name}-1.tar.gz \
+      -C ${name} --strip-components=1
+
+    sourceRoot=${name}
+  '';
+
+  buildInputs = [
+    autoconf
+    automake
+    glib
+    gnome2.gtk
+    gnome2.libglade
+    libtool
+    pkgconfig
+
+    cndrvcups-common
```

Some build tools here should go to `nativeBuildInputs`:
 
```suggestion
    nativeBuildInputs = [ autoconf automake libtool pkgconfig ];
```